config/500-controller.yaml

```diff
@@ -52,6 +52,9 @@ spec:
         ports:
           - containerPort: 9090
             name: metrics
+        resources:
+          limits:
+            memory: 1000Mi
```

Comment:
Can you add a comment explaining the provenance of this value? Even if it's just `dart board`, that gives the future maintainer context about how the value was derived and what would be required to improve on it.

Reply:
Done - see if my comment is sufficient.